app/models/manageiq/providers/cloud_manager/provision_workflow.rb

```diff
@@ -31,6 +23,12 @@ def allowed_cloud_subnets(_options = {})
     end
   end
 
+  def allowed_cloud_networks(_options = {})
+    source = load_ar_obj(get_source_vm)
+    targets = get_targets_for_ems(source, :cloud_filter, CloudNetwork, 'cloud_networks')
```

Comment:
@d-m-u - I think you may just need to use `get_targets_for_source` here as opposed to the `_ems` variant.

The original code iterated off the `src_obj` without the need to lookup the `ems` so you shouldn't need to lookup the `ems` with this fix as well.